### ISSUE TITLE
Tab order does not work in WASAPI interface to traverse between fields

### ISSUE THREAD

**Issue by machawk1** [machawk1/wail#507]
This comes down to not using a wx.Panel and instead adding the elements to the frame. wxPython's tab traversal code is in Panel so without a panel, the expected behavior will not be exhibited.

**Comment by machawk1** [comment]
Above approach, merged into issue-355 (wasapi) branch.
